tool/src/main/java/org/wildfly/security/tool/CredentialStoreCommand.java

```diff
@@ -50,12 +64,11 @@
 import org.apache.commons.cli.Option;
 import org.apache.commons.cli.OptionGroup;
 import org.apache.commons.cli.Options;
-
 import org.apache.sshd.common.config.keys.FilePasswordProvider;
 import org.apache.sshd.common.config.keys.PublicKeyEntry;
 import org.wildfly.common.iteration.CodePointIterator;
-import org.wildfly.security.auth.util.ElytronFilePasswordProvider;
 import org.wildfly.security.auth.server.IdentityCredentials;
+import org.wildfly.security.auth.util.ElytronFilePasswordProvider;
```

Comment:
It's good to avoid reordering imports since this type of change isn't related to the PR.

Reply:
ah, eclipse formatter kicking in, will double check in all commits